core/src/main/java/com/predic8/membrane/core/interceptor/HTTPClientInterceptor.java

```diff
@@ -53,8 +54,12 @@ public HTTPClientInterceptor() {
     }
 
     @Override
-    public Outcome handleRequest(Exchange exc) throws Exception {
-        exc.blockRequestIfNeeded();
+    public Outcome handleRequest(Exchange exc) {
+        try {
+            exc.blockRequestIfNeeded();
+        } catch (TerminateException e) {
+            log.error("Could not block request: {}",e.getMessage(), e);
```

Comment:
e.getMessage() is already printed, because `e` is passed to `log.error(...)` in the `Throwable` parameter.